Add Helm instructions to the docs (#2508)
benthosdev/benthos: website/docs/guides/getting_started.md

docker run --rm -v /path/to/your/config.yaml:/benthos.yaml ghcr.io/benthosdev/benthos
```

### Helm

> If you're using Kubernetes, you can install Benthos using [Helm](https://helm.sh/). You can find more information on configuring the Helm Chart, [here](https://github.com/benthosdev/benthos-helm-chart/).

Reviewer: ```suggestion
If you're using Kubernetes, you can install Benthos using [Helm](https://helm.sh/). You can find more information on configuring the Helm Chart, [here](https://github.com/redpanda-data/helm-charts/tree/main).
```

```sh

> helm repo add benthos https://benthosdev.github.io/charts/

Reviewer: ```suggestion
helm repo add redpanda https://charts.redpanda.com/
```

helm repo update

> helm install benthos/benthos

Reviewer: ```suggestion
helm install redpanda/connect
```
